### PR TITLE
fix: the previous cop is completely overwritten when binding op

### PR DESCRIPTION
**Pull request by qclc**

### CODE REVIEW

File: pkg/controllers/override/util.go [hunk 1]
```diff
@@ -112,12 +113,37 @@ func lookForMatchedPolicies(
...
+	if err != nil {
+		return nil, fmt.Errorf("failed to get sourceObj from fedObj: %w", err)
+	}
+	return &helpData{
+		gvk:       gvk,
+		sourceObj: sourceObj,
+	}, nil
+}
+
+// 集群名称，和对应的所有patches
```
Comment (JackZxj) [inline pkg/controllers/override/util.go:136]: please translate it

File: pkg/util/pod/pod.go [hunk 2]
```diff
@@ -76,3 +76,24 @@ func GetResourcePodSpec(fedObject fedcorev1a1.GenericFederatedObject, gvk schema
 	}
 	return GetPodSpec(fedObject, path)
 }
+
+func GetResourcePodSpecFromUnstructuredObj(
```
Comment (JackZxj) [inline pkg/util/pod/pod.go:80]: nit: The above func GetPodSpec can reuse this to simplify

File: pkg/controllers/override/util.go (hunk 1 above)
Comment (qclc) [reply pkg/controllers/override/util.go]: sorry, this content is a comment for my own viewing only and has been deleted.

File: pkg/util/pod/pod.go (hunk 2 above)
Comment (qclc) [reply pkg/util/pod/pod.go:84]: The common logic of this function has been extracted.

**Review by JackZxj — APPROVED** [review]

**Review by mrlihanbo — APPROVED** [review]

**Pull request merged**